spec/api/viewing_all_applications_spec.rb

```diff
@@ -0,0 +1,34 @@
+# TODO: import this automagically for /api folder
```

Comment:
Can we put this in `spec/requests/api` as it is a request spec?

Reply:
Addressed in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/33/commits/02d0bb60a7ae90c7b05e910b2759b155903db2e3